test/snippet/core/detail/customisation_point.cpp

```diff
@@ -0,0 +1,92 @@
+#include <seqan3/std/concepts>
+#include <vector>
+
+#include <seqan3/core/detail/customisation_point.hpp>
+
+//! [CPO Definition]
+namespace seqan3::detail::adl_only
+{
+// Poison-pill overload to prevent non-ADL forms of unqualified lookup.
+template <typename ...args_t>
+void begin(args_t ...) = delete;
+
+struct begin_cpo : public detail::customisation_point_object<begin_cpo, 1>
+{
+    using base_t = detail::customisation_point_object<begin_cpo, 1>;
+    // Only this class is allowed to import the constructors from base_t. (CRTP safety idiom)
+    using base_t::base_t;
+
+    // range.begin(), member access
+//! [SEQAN3_CPO_OVERLOAD]
```

Comment:
Here is a snippet in a snippet (this comment line is visible in another snippet), is this intended?

Reply:
No! It isn't they will be filtered out, see https://seqan3-git-fork-marehr-newcpo-seqan.vercel.app/dev/html/structseqan3_1_1detail_1_1customisation__point__object.html#details

I always thought that this is the case, but I guess doxygen changed that?

Reply:
Great, I like this new feature!